File: Resource/51900286-51900712-51900718-51900699/source demo/Window Form/51900286-51900712-51900718-51900699/RestaurentDTO/TaiKhoan.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RestaurentDTO
{
    public class TaiKhoan
    {
        private string _tenTaiKhoan;
        private string _matKhau;
        private string _maNhanVien;

        public TaiKhoan(string tenTaiKhoan, string matKhau, string maNhanVien)
        {
            TenTaiKhoan = tenTaiKhoan;
            MatKhau = matKhau;
            MaNhanVien = maNhanVien;
        }

        public TaiKhoan( DataRow dataRow)
        {
            TenTaiKhoan = dataRow["TenTaiKhoan"].ToString();
            MatKhau = dataRow["MatKhau"].ToString();
            MaNhanVien = dataRow["MaNhanVien"].ToString();
        }

        public TaiKhoan()
        {
            TenTaiKhoan = "";
            MatKhau = "";
            MaNhanVien = "";
        }

        public string TenTaiKhoan { get => _tenTaiKhoan; set => _tenTaiKhoan = value; }
        public string MatKhau { get => _matKhau; set => _matKhau = value; }
        public string MaNhanVien { get => _maNhanVien; set => _maNhanVien = value; }
    }
}

File: Dev/Models/Models/TaiKhoan.java
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;


@Entity
@ToString
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@Builder
@Table(name="taikhoan")
public class TaiKhoan {
    @Id
    @GeneratedValue
    private Long id;
    private String TenTaiKhoan;
    private String MatKhau;
    private String MaNhanVien;
}